kb/2025-11-03/vm-slowness-issue.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Harvester VM Network Slowness – GRO/GSO Offload Issue
+description: 'Reason for VM Network slowness and how to fix it by disabling GRO/GSO offload'
```

Comment:
'Reason for VM Network slowness and how to fix it by disabling GRO/GSO offload'
->
'Reason for VM Network slowness and how to fix it by disabling GRO/GSO offload caused by some specific NICs'